game/src/layer/parking.rs

```diff
@@ -95,6 +95,57 @@ impl Occupancy {
         private_bldgs: bool,
         looking_for_parking: bool,
     ) -> Occupancy {
+        let mut total_ppl = 0;
+        let mut has_car = 0;
+        for p in app.primary.sim.get_all_people() {
+            total_ppl += 1;
+            if p.vehicles
+                .iter()
+                .any(|v| v.vehicle_type == VehicleType::Car)
+            {
+                has_car += 1;
+            }
+        }
+
+        if app.primary.sim.infinite_parking() {
+            let panel = Panel::new(Widget::col(vec![
```

Comment:
Why is this panel only relevant when `infinite_parking` is enabled?

Reply:
To answer my own question, there is a *separate* parking occupancy panel for non-infinite parking.